node/src/BaseClient.ts

```diff
@@ -1231,6 +1786,20 @@ export class BaseClient {
      * @returns the number of members removed.
      * If `key` does not exist, it is treated as an empty sorted set, and the command returns 0.
      * If `minScore` is greater than `maxScore`, 0 is returned.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of zremRangeByScore method to remove members from a sorted set based on score range
+     * const result = await client.zremRangeByScore("my_sorted_set", \{ bound: 5.0, isInclusive: true \}, "positiveInfinity");
```

Comment:
Isn't there a constant for `positiveInfinity`?

Reply:
not something that I know of